spec/models/student_spec.rb

```diff
@@ -13,4 +14,5 @@
     it {should have_many(:professors).through(:professor_students)}
   end
 
+
```

Comment:
To test the average age of the student I would keep it simple and do something like this:

```
it 'calculates the average age of students' do
    harry = Student.create(name: "Harry Potter" , age: 11 , house: "Gryffindor" )
    malfoy = Student.create(name: "Draco Malfoy" , age: 12 , house: "Slytherin" )
    longbottom = Student.create(name: "Neville Longbottom" , age: 11 , house: "Gryffindor" )
    average_age = ([harry.age, malfoy.age, longbottom.age].sum)/3

    expect(Student.average_student_age.round(0)). to eq(average_age)
  end
```

I'm curious what approach you were taking that lead you down the path of running into a no method error?